turbo.json

```diff
@@ -117,6 +109,19 @@
     "build:readme": {
       "outputs": ["dist/**/*.md"]
     },
+    "build:types": {
```

Comment:
I added this one here because it does seem likely to me that other packages may need to implement this one (e.g. maybe we'll decide to use swc to do the build in more places)